package/boot/kexec-tools/patches/100-kexec-tools-fix-for-Unhandled-rela-relocation-R_X86_64_PLT32-error.patch

```diff
@@ -1,8 +1,6 @@
-diff --git a/kexec/arch/x86_64/kexec-elf-rel-x86_64.c b/kexec/arch/x86_64/kexec-elf-rel-x86_64.c
-index 7fdde73..af33689 100644
 --- a/kexec/arch/x86_64/kexec-elf-rel-x86_64.c
 +++ b/kexec/arch/x86_64/kexec-elf-rel-x86_64.c
-@@ -78,7 +78,8 @@ void machine_apply_elf_rel(struct mem_ehdr *UNUSED(ehdr),
+@@ -78,7 +78,8 @@ void machine_apply_elf_rel(struct mem_eh
```

Comment:
Entirely offtopic here, but I wonder whether we should do something about these abbreviations.
The causes additional warnings during checkpatch, and the actually quite regularly destroy information since the differences are frequently at the end of the line (and thus cut off).

Reply:
IMO, no. All packages in the packages feed have gone through this with no ill effects.